apps/website/src/components/LiveExample.tsx

```diff
@@ -15,7 +15,11 @@ import { createRoot } from 'react-dom/client';
 import { ThemeProvider } from '@itwin/itwinui-react';
 import App from './App';
 import './styles.css';
-createRoot(document.getElementById('root')).render(<ThemeProvider theme='dark'><App /></ThemeProvider>);
+createRoot(document.getElementById('root')!).render(
```

Comment:
Does the `!` have something to do with null safety? Just curious about why is it needed.

Reply:
yes, try it in a codesandbox with and without `!`
https://vite.new/react-ts